client/src/components/Onboarding/OnboardingAdmin.js

```diff
@@ -132,37 +131,33 @@ const OnboardingAdminListUser = ({ user, viewLogCallback }) => {
   const adminAction = useSelector(state => state.onboarding.action);
 
   return (
+    /* !!!: Temporary extra markup to make simpler PR diff */
+    /* !!!: Temporary bad indentation markup to make simpler PR diff */
+    /* eslint-disable prettier/prettier */
+    <>
+    {user.steps.map((step, index) => (
     <tr styleName="user">
```

Comment:
I'm seeing some more odd layout for multiple users
<img width="1550" alt="Screen Shot 2021-03-17 at 11 37 24 AM" src="https://user-images.githubusercontent.com/20326896/111504357-904d9a80-8715-11eb-9438-99994f6eb452.png">

Reply:
> I'm seeing some more odd layout for multiple users
> <img alt="Screen Shot 2021-03-17 at 11 37 24 AM" width="1550" src="https://user-images.githubusercontent.com/20326896/111504357-904d9a80-8715-11eb-9438-99994f6eb452.png">

@rstijerina what's the second user record on your system?

Reply:
Seeing the same issue, even after bringing down volumes. Same user repeated twice.

Reply:
fixed in 5f3ee56!